modules/lambdas/main.tf

```diff
@@ -89,6 +89,34 @@ resource "aws_lambda_function" "delete_old_reconcile_jobs" {
   }
 }
 
+# rule to run the lambda every day
+resource "aws_cloudwatch_event_rule" "delete_old_reconcile_jobs_daily_event_rule" {
+  ## REQUIRED
+  name                = "${var.prefix}_delete_old_reconcile_jobs_daily_event_rule"
```

Comment:
possibly remove `daily` from the name to make more generic

Reply:
Fixed